### ISSUE TITLE
Support both passcode and TouchID at the same time like iphone passcode lockscreen

### ISSUE THREAD

**Issue by manhcuong27** [yankodimitrov/SwiftPasscodeLock#74]
Hello,

Just wonder if this possible, it is a bit annoy when every time user open app, a touchID pop up appear

Thanks.

**Comment by ziogaschr** [comment]
Yes, it is possible. I don't remember if I have added this in the following PR https://github.com/velikanov/SwiftPasscodeLock/pull/27 as an example.

I don't have time at the moment to check, but if not, ping me and I will try to give you some code or at least some directions.

**Comment by vadimvitvickiy** [comment]
Hello, I'm interested in this solution too
